frontend/pages/beta/model/[modelId].tsx

```diff
@@ -30,9 +28,7 @@ export default function Model() {
             { title: 'Releases', view: <Releases model={model} /> },
             { title: 'Settings', view: <Settings model={model} /> },
           ]}
-          actionButtonOnClick={myFunction}
-          actionButtonTitle='Actions'
-          displayActionButton
+          displayActionButton={false}
```

Comment:
It's unusual to provide `false` as a prop. Consider not providing displayActionButton at all here. If `displayActionButton` isn't optional, it probably should be.

Reply:
It is optional already so not sure why I was adding in false here!